_posts/2021-07-07-cockpit-248.md

```diff
@@ -0,0 +1,67 @@
+
+![screenshot of install missing packages]({{ site.baseurl }}/images/248-metrics-install-missing-packages.png)
+
+![screenshot of install missing packages]({{ site.baseurl }}/images/248-metrics-install-missing-packages-2.png)
+
+This completes the story of [setting up PCP and Grafana metrics with Cockpit](https://cockpit-project.org/blog/pcp-grafana.html).
+
+## PAM: Deprecate `pam_cockpit_cert` module
+
+The `pam_cockpit_cert` module is deprecated and no longer required for client certificate authentication. Cockpit continues to ship a stub module which returns `PAM_IGNORE`, but a future release will drop this stub.
+
+Upgrading packages will not update a modified `/etc/pam.d/cockpit` file. If you have modified this file, please take care to remove the relevant entry.
+
+## Machines: Bug fixes and improvements
+
+Cockpit-machines has a new release with several bug fixes and minor improvements.
+
+## OSTree: Bump epoch to 1
+
+The cockpit-ostree RPM package is being released as version `1:186`, with a newly introduced epoch. This new version will re-enable upgrading from an unintentional "version 999" of cockpit-ostree, which was accidentally published while testing the release infrastructure.
+
+## Developers: `main` branch switch
+
+Over the past few months, many Cockpit sub-projects changed their primary development branch to `main`. As of this release, all Cockpit repositories&mdash;including the main Cockpit repository&mdash;have finished the switch. Developers with checked out git repositories should [update their branches](https://docs.github.com/en/github/administering-a-repository/managing-branches-in-your-repository/renaming-a-branch#updating-a-local-clone-after-a-branch-name-changes). 
+
+This set of releases also marks the end of our transition to Dart Sass and Webpack 5, as well as cleanups and modernization to Cockpit's build system.
+
+## Try it out
+
+Cockpit 248, cockpit-machines 247, and cockpit-ostree 187 are available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/248)
+* [Cockpit Fedora 34](https://bodhi.fedoraproject.org/updates/FEDORA-2021-56a2a973ac)
+* [Cockpit Fedora 33](https://bodhi.fedoraproject.org/updates/FEDORA-2021-c38f16f21e)
+* [Cockpit-machines Source Tarball](https://github.com/cockpit-project/cockpit-machines/releases/tag/247)
+* [Cockpit-machines Fedora 34](https://bodhi.fedoraproject.org/updates/FEDORA-2021-8a0b90aea9)
+* [Cockpit-machines Fedora 33](https://bodhi.fedoraproject.org/updates/FEDORA-2021-eee63f130a)
+* [Cockpit-ostree Source Tarball](https://github.com/cockpit-project/cockpit-ostree/releases/tag/187)
```

Comment:
404.  Should be 186.

Reply:
It's because I'm a human editing this and not a script preparing it. :robot: